Examples/Objective-C/ViewController.m

```diff
@@ -142,8 +142,14 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 }
 
 - (void)startNavigation:(MBRoute *)route {
+    // Using storyboard
     [[MBNavigationUI shared] setTintColor:[UIColor redColor]];
```

Comment:
Good idea but I think that should be part of a bigger overhaul. Expose a style property on RouteViewController that grabs the tint of the parent instead of this shared instance.